pyafq_cli.json

```diff
@@ -0,0 +1,501 @@
+        "name": "scalars",
+        "type": "String",
+        "list": true,
+        "min-list-entries": 1,
+        "list-separator": "', '",
+        "value-key": "[SCALARS]",
+        "value-choices": ["dti_fa", "dti_md"],
+        "default-value": ["dti_fa", "dti_md"],
+        "optional": true
+    },
+    {
+        "id": "directions",
+        "name": "directions",
+        "type": "String",
+        "value-key": "[DIRECTIONS]",
+        "value-choices": ["det", "prob"],
+        "default-value": "det",
+        "optional": true
+    },
+    {
+        "id": "max_angle",
+        "name": "max angle",
+        "type": "Number",
+        "value-key": "[MAX_ANGLE]",
+        "minimum": 0.0,
+        "maximum": 180.0,
+        "default-value": 30.0,
+        "optional": true
+    },
+    {
+        "id": "sphere",
+        "name": "sphere",
+        "type": "String",
+        "value-key": "[SPHERE]",
+        "default-value": "None",
+        "optional": true
+    },
+    {
+        "id": "seed_mask",
+        "name": "seed mask",
+        "type": "String",
+        "value-key": "[SEED_MASK]",
+        "default-value": "None",
+        "optional": true
+    },
+    {
+        "id": "n_seeds",
+        "name": "n seeds",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[N_SEEDS]",
+        "minimum": 1,
+        "default-value": 1,
+        "optional": true
+    },
+    {
+        "id": "random_seeds",
+        "name": "random seeds",
+        "type": "String",
+        "value-key": "[RANDOM_SEEDS]",
+        "value-choices": ["false", "true"],
+        "default-value": "false",
+        "optional": true
+    },
+    {
+        "id": "stop_mask",
+        "name": "stop mask",
+        "type": "String",
+        "value-key": "[STOP_MASK]",
+        "default-value": "None",
+        "optional": true
+    },
+    {
+        "id": "stop_threshold",
+        "name": "stop threshold",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[STOP_THRESHOLD]",
+        "minimum": 0,
+        "default-value": 0,
+        "optional": true
+    },
+    {
+        "id": "step_size",
+        "name": "step size",
+        "type": "Number",
+        "value-key": "[STEP_SIZE]",
+        "minimum": 0.0,
+        "default-value": 0.5,
+        "optional": true
+    },
+    {
+        "id": "min_length",
+        "name": "min length",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[MIN_LENGTH]",
+        "minimum": 1,
+        "default-value": 10,
+        "optional": true
+    },
+    {
+        "id": "max_length",
+        "name": "max length",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[MAX_LENGTH]",
+        "default-value": 1000,
+        "optional": true
+    },
+    {
+        "id": "odf_model",
+        "name": "odf model",
+        "type": "String",
+        "value-key": "[ODF_MODEL]",
+        "value-choices": ["DTI"],
```

Comment:
Similar to `scalars_model`, are the options here `DTI` and `DKI`?

Reply:
Confusingly, it's "DTI" and "CSD" for this one.